ui/scripts/configuration.js

```diff
@@ -2926,6 +2926,270 @@
                 }
             },
 
+            backupOfferings: {
+                type: 'select',
+                title: 'label.menu.backup.offerings',
+                listView: {
+                    id: 'backupOfferings',
+                    label: 'label.menu.backup.offerings',
+                    fields: {
+                        name: {
+                            label: 'label.name',
+                            editable: true
+                        },
+                        description: {
+                            label: 'label.description'
+                        },
+                        zoneid: {
```

Comment:
Displaying the zone name will be better IMO, it is already available on the response of the backup offering listing

Reply:
Fixed.